samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/logic/base/AbstractComponentFacade.java

```diff
@@ -55,8 +54,8 @@ public AbstractComponentFacade() {
 
   /**
    * Creates a {@link Map} with all {@link GenericEntity entities} from the given {@link Collection} using their
-   * {@link GenericEntity#getId() ID} as key. All {@link GenericEntity entities} without an
-   * {@link GenericEntity#getId() ID} ({@code null}) will be ignored.
+   * {@link GenericEntity#getId() ID} as key. All {@link GenericEntity entities} without an {@link GenericEntity#getId()
```

Comment:
Also selbst auf einer frischen Umgebung werden die Änderungen so durchgeführt. In der `AbstractComponentFacade` z.B. werden die io.oasp imports nach unten geschoben und die Zeilenlänge in Zeile 58/59 "gekürzt". Das sind genau die Einstellungen die wohl bei der oasp-IDE-2.0 mitgeliefert werden (mit zwei Kollegen getestet - bei allen ändern sich Import und Zeilenlänge beim Speichern der Klasse).
Entweder werden alle Klassen an diese Einstellung angepasst, oder die Config wird abgeändert. Im Code finden sich momentan beide Varianten - da sollte man eine Lösung finden: #454